src/main/java/core/basesyntax/service/activitiy/AddingHandler.java

```diff
@@ -5,7 +5,11 @@
 import core.basesyntax.model.FruitCrate;
 
 public class AddingHandler implements ActivityHandler {
-    private static final StorageDao storageDao = new StorageDaoCsvImpl();
+    private final StorageDao storageDao;
+
+    public AddingHandler() {
+        this.storageDao = new StorageDaoCsvImpl();
+    }
```

Comment:
you should pass the implementation TO the constructor (from main or from another place where you create an AddingHandler instance)